src/Eluceo/iCal/Component/Event.php

```diff
@@ -197,7 +197,7 @@ class Event extends Component
 
     /**
      * Dates to be excluded from a series of events.
-     * 
+     *
```

Comment:
There has been an empty space here that was being kicked by the editor on saving.